resources/styles.css

```diff
@@ -73,12 +106,6 @@
     cursor: default;
 }
 
-.open a.edit-button.tap-target-origin, .open a.edit-button.tap-target-origin:hover {
```

Comment:
Why is this removed? I assume is old code, but I'm not sure what was this for

Reply:
Not sure. Must have happened when I merged or something.

Reply:
please check what is this for

Reply:
I really have no idea, so I'll just put it back in.